auth_session_timeout/models/ir_config_parameter.py

```diff
@@ -5,15 +5,14 @@
 
 from openerp import models, api, tools, SUPERUSER_ID
 
-
 DELAY_KEY = 'inactive_session_time_out_delay'
 IGNORED_PATH_KEY = 'inactive_session_time_out_ignored_url'
 
 
 class IrConfigParameter(models.Model):
     _inherit = 'ir.config_parameter'
 
-    @tools.ormcache(skiparg=0)
+    @tools.ormcache('db')
     def get_session_parameters(self, db):
```

Comment:
@chienandalu 
Glad you found my changes useful :)
Yes I struggled with the same thoughts myself - at first I couldn't see any reason why changes were required at all in ir_config_parameter.
The best I could come up with is that the original writer wanted to read both and cache once, rather than read twice and cache twice.
In any event it is indeed redundant, although I was advised against removing get_session_parameters (deceptively named for a public method :/) as its in the wild now and might be referenced by some unknown modules.